### ISSUE TITLE
Pressing "Return" key at end of ordered list doesn't work as expected.

### ISSUE THREAD

**Issue by themightymo** [WordPress/gutenberg#5122]
## Issue Overview
When I create an ordered list and I'm done with the list, I hit "Return".  In gmail, hitting return twice removes the last item and brings me to the next line.  This is great.  

But in Gutenberg, hitting return doesn't do anything - it stays on the last ordered list item.

<img width="821" alt="gutenberg ordered list issue" src="https://user-images.githubusercontent.com/179105/36352906-0d9ce218-1485-11e8-876d-74c8f6ef1224.png">

## Steps to Reproduce (for bugs)
1. Create an ordered list
2. Add a few items to the list
3. Hit "Return" key.

WP 4.9.4
Gutenberg 2.2.0

## Expected Behavior
Pressing "Return" should remove this item and bring me to the next block.

## Current Behavior
Pressing "Return" does nothing.

**Comment by rianrietveld** [comment]
This can be closed, is duplicate of #5576
